packages/kbn-management/cards_navigation/src/types.ts

```diff
@@ -34,9 +67,33 @@ export interface ManagementAppProps {
 }
 
 export interface AppDefinition {
-  category: string;
+  category: typeof appCategories[keyof typeof appCategories];
   description: string;
-  icon: React.ReactElement;
+  icon: EuiIconProps['type'];
 }
 
-export type AppProps = ManagementAppProps & AppDefinition;
+export type CardNavExtensionDefinition = AppDefinition &
```

Comment:
Could this type be exported from the package index? And the record type `Record<string, CardNavExtensionDefinition>` as well, so we can be more strict. We will need this type from Security to define the `extendCardNavDefinitons`.

Reply:
I do understand the argument for readability, I should point out we can still use the type definition from the component props that's already exposed.